unittests/Feature/FeatureModelWriter.cpp

```diff
@@ -8,7 +8,7 @@
 
 namespace vara::feature {
 
-const std::string TESTFILEDIRPATH = "../../../test/";
```

Comment:
Move this a separate test header similar to https://github.com/se-passau/VaRA/blob/vara-dev/unittests/UnittestHelper.h.